app/javascript/components/mappers/domainToApi/application.js

```diff
@@ -31,12 +30,12 @@ export const applicationFieldMapper = {
   reserved_senior: 'hasSenior',
   application_submission_type: 'applicationSubmissionType',
   processing_status: 'processingStatus',
-  householdAssets: (source) => currencyToFloat(source.household_assets),
```

Comment:
I'm pretty sure that with these you can just actually flip it to match e.g. line 33 above, no need for the lambda fn

Reply:
updated